src/core/files.ml

```diff
@@ -266,16 +266,7 @@ let file_to_module : string -> Path.t = fun fname ->
       fatal_no_pos "Expected any of: %a." pp_exp valid_extensions
     end;
   (* Normalizing the file path. *)
-  let fname =
-    let rec normalize fname =
-      if Sys.file_exists fname then
-        Filename.realpath fname
-      else
-        let dirnm = Filename.dirname fname in
-        let basenm = Filename.basename fname in
-        Filename.concat (normalize dirnm) basenm
-    in
-    normalize fname
+  let fname = Filename.normalize fname
   in
```

Comment:
Thanks to put the "in" on the same line as "let" as everything holds on a single line in this case.

Reply:
done